pkgs/build-support/node/fetch-yarn-deps/yarn-install-hook.sh

```diff
@@ -17,7 +17,7 @@ yarnInstallHook() {
         --offline
     )
 
-    local -r tmpDir="$(mktemp -d yarnInstallHook.XXXXXX)"
```

Comment:
Thanks for catching that. I wonder why we used a temporary directory at all, but I don't mind that much now.